packages/website/src/hooks/cannon.ts

```diff
@@ -76,6 +76,7 @@ export function useCannonBuild(safe: SafeDefinition | null, def?: ChainDefinitio
   const { addLog } = useLogs();
   const settings = useStore((s) => s.settings);
 
+  const [finishedBuilding, setFinishedBuilding] = useState(false);
   const [isBuilding, setIsBuilding] = useState(false);
```

Comment:
lets collapse `isBuilding` and `finishedBuilding` into a single state `status` which is a string that can be of value `building`|`built`|`idle` or whatever

the booleans can still exist, but they should be computed from this one state